build.gradle

```diff
@@ -46,6 +46,19 @@ if (!hasProperty('buildProfile') || (hasProperty('buildProfile') && buildProfile
 	targetCompatibility = 22
 }
 
+if (!hasProperty('buildProfile') || (hasProperty('buildProfile') && buildProfile == "jre21")) {
```

Comment:
Please correct me if I am wrong but isn't (!hasProperty('buildProfile') || (hasProperty('buildProfile') always true?

Reply:
Sorry ... missed the order of brackets. Never mind